src/script.js

```diff
@@ -161,6 +176,7 @@ function isCanonicalSignature (buffer) {
 module.exports = {
   compile: compile,
   decompile: decompile,
+  toWitness: toWitness,
```

Comment:
maybe we should just have `bitcoin.witness` and have `compile/decompile` there instead?